src/oauth.js

```diff
@@ -41,7 +41,8 @@
     'oauth.dribble',
     'oauth.pocket',
     'oauth.mercadolibre',
-    'oauth.xing'])
+    'oauth.xing',
```

Comment:
I'm not sure where oauth.xing came from. I may have started on the master branch or something.